spec/lib/appsignal/integrations/http_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+if DependencyHelper.http_present?
+  require "appsignal/integrations/http"
+
+  describe Appsignal::Integrations::HttpIntegration do
+    around do |example|
+      keep_transactions { example.run }
+    end
+
+    before :context do
+      start_agent
+    end
+
+    it "should instrument a HTTP request" do
+      stub_request(:get, "http://www.google.com/")
+
+      expect { HTTP.get("http://www.google.com") }
+        .to change { created_transactions.length }.by(1)
+
+      expect(last_transaction).to be_completed
+      expect(last_transaction.to_h).to include("namespace" => Appsignal::Transaction::HTTP_REQUEST)
```

Comment:
```suggestion
      expect(transaction).to be_completed
      expect(transaction.to_h).to include("namespace" => Appsignal::Transaction::HTTP_REQUEST)
```

Use the transaction created at the start of the spec, see my other comment.